rkt/rkt.go

```diff
@@ -165,7 +216,7 @@ func garbageDir() string {
 }
 
 func getKeystore() *keystore.Keystore {
-	if globalFlags.InsecureSkipVerify {
+	if globalFlags.InsecureFlags.SkipImageCheck() {
```

Comment:
I think we can remove this. See https://github.com/coreos/rkt/commit/be3b49e670ccb5d10c20202b8161074578f3cd14

We don't decide if we're checking the signature if the keystore is null anymore.